dockstore-common/src/test/java/io/dockstore/common/yaml/DockstoreYamlTest.java

```diff
@@ -198,6 +198,32 @@ void testMissingPrimaryDescriptor() {
         }
     }
 
+    @Test
+    void testRelativePrimaryDescriptor() {
+        try {
+            final String content = DOCKSTORE12_YAML.replaceAll("primaryDescriptorPath: /", "primaryDescriptorPath: ");
+            DockstoreYamlHelper.readDockstoreYaml(content, true);
+            fail("Invalid dockstore.yml not caught");
+        } catch (DockstoreYamlHelper.DockstoreYamlException e) {
+            // check that the error message contains the name of the property and an appropriate adjective
+            assertTrue(e.getMessage().contains("primaryDescriptor"));
+            assertTrue(e.getMessage().matches(".*(the path must be an absolute path to be valid)*"));
```

Comment:
Could use `.contains` like above to verify that it contains `the path must be an absolute path to be valid` instead of using a regex